CITATIONS.md

```diff
@@ -10,8 +10,16 @@
 
 ## Pipeline tools
 
+- [BCFtools](https://academic.oup.com/gigascience/article/10/2/giab008/6137722)
+
+  > Danecek P, Bonfield JK, Liddle J, et al. Twelve years of SAMtools and BCFtools. GigaScience. 2021;10(2):giab008. doi:10.1093/gigascience/giab008
+
```

Comment:
```suggestion
- [BCFtools](https://academic.oup.com/gigascience/article/10/2/giab008/6137722)

  > Danecek P, Bonfield JK, Liddle J, et al. Twelve years of SAMtools and BCFtools. GigaScience. 2021;10(2):giab008. doi:10.1093/gigascience/giab008

- [SAMtools](https://academic.oup.com/gigascience/article/10/2/giab008/6137722)

  > Danecek P, Bonfield JK, Liddle J, et al. Twelve years of SAMtools and BCFtools. GigaScience. 2021;10(2):giab008. doi:10.1093/gigascience/giab008
 
```

Reply:
Just wanted to add SAMtools too and this was the easiest way, as the paper and link are the same